### ISSUE TITLE
FR: Perception expander: prevent burn in by shifting a few pixels basdd on time

### ISSUE THREAD

**Issue by KopiasCsaba** [koreader/koreader#12696]
**Does your feature request involve difficulty completing a task? Please describe.**
no

**Describe the solution you'd like**
Some settings to enable timely horizontal shifting of the lines.

E.g. the lines would shift back and forth in a 10px wide area, e.g. every 5 mins it would shift one pixel. Or smtg like this.

**Comment by pazos** [comment]
Burn in where?. AFAICT that's something affecting oled and black pixels are powered off leds in there.

If you use oled and you're worry about burn in I would suggest to avoid the program entirely. Perception expander shouldn't be an issue. The white borders of status bar elements should.

**Comment by KopiasCsaba** [comment]
I have meant it for eink screens. Yeah not using it is also an option i agree, not the biggest deal :)

**Comment by NiLuJe** [comment]
Yeah, burn in isn't a thing on eInk.

**Comment by Commodore64user** [comment]
> Yeah, burn-in isn't a thing on eInk.

One of my kindle keyboards begs to disagree. Perhaps it might technically not be burn-in but, mine definitely has a line across the top of screen (the separator one gets in the library mode, stock firmware) that will not stop being there.

**Comment by KopiasCsaba** [comment]
Yeah I have seen it in stores where many devices are out on the shelf to play around. But i understand if its not the most burning (pun intended) issue. Maybe once upon a time.

**Comment by NiLuJe** [comment]
The K3 (and possibly other similar Pearl-era devices) have a specific issue where they adversely react to UV (well, more than expected, at least ;p) and it can lead to weirdness (usually, loss of contrast).
